pkgs/development/haskell-modules/configuration-darwin.nix

```diff
@@ -382,6 +382,9 @@ self: super: ({
     '';
   }) super.di-core;
 
+  # Requires /usr/bin/security which breaks sandbox
+  servant-auth-server = dontCheck super.servant-auth-server;
+
```

Comment:
Morally this needs `__darwinAllowLocalNetworking = true;`, but since the tests only build outside of the sandbox anyway it has no effect.